creusot/src/ctx.rs

```diff
@@ -97,6 +98,7 @@ pub struct TranslationCtx<'tcx> {
     sig: HashMap<DefId, PreSignature<'tcx>>,
     bodies: HashMap<LocalDefId, BodyWithBorrowckFacts<'tcx>>,
     opacity: HashMap<DefId, Opacity>,
+    pub(crate) span_map: SpanMap,
```

Comment:
Perhaps we should move this to the `Why3Generator`? My (currently still imperfect) rule of thumb is that `TranslationCtx` should be Why3 independent, handling the "Rust -> Creusot" portion while Why3Generator does the "Creusot -> Why3" bit.